src/renderer/base/renderer.cpp

```diff
@@ -195,9 +270,6 @@ try
     // 2. Paint Rows of Text
     _PaintBufferOutput(pEngine);
 
-    // 3. Paint overlays that reside above the text buffer
-    _PaintOverlays(pEngine);
-
     // 4. Paint Selection

```

Comment:
I'd prefer not changing this code right now, because I'd like to remove it in the near-ish term and changing it now would only disturb the `git blame` IMHO.